lightbluetent/admin.py

```diff
@@ -1,39 +1,194 @@
-from flask import Blueprint, render_template, request, flash
-from lightbluetent.models import db
+import re
+
+from flask import Blueprint, render_template, request, flash, abort, redirect, url_for
+from lightbluetent.models import db, User, Society
+from lightbluetent.utils import auth_decorator, validate_email
+
+ILLEGAL_NAME_RE = re.compile(r'[:,=\n]')
+ILLEGAL_NAME_ERR = "Please do not use any of the following characters: : , = ⏎ "
```

Comment:
I can see why newlines should be disallowed, but why the rest?

Reply:
Again, just taken from control-panel: will remove if everyone's happy to.

Reply:
Ah, I see -- the reason we have those in control-panel is so that the [GECOS field](https://en.wikipedia.org/wiki/Gecos_field) in /etc/passwd doesn't receive invalid data.  It can definitely go here.